tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py

```diff
@@ -7,43 +7,196 @@
 from gaiatest.apps.settings.app import Settings
 
 
-class TestSettingsRTL(GaiaImageCompareTestCase):
+class TestSettingsRTLNC(GaiaImageCompareTestCase):
     def test_settings_app(self):
 
         settings = Settings(self.marionette)
         settings.launch()
 
-        # opening each subpage in Settings menu.  Privacy Control is not opened, because it is a separate app
-        #some subpages have their own subpages, and they need to be opened as well.
-        settings.open_wifi()
-        self.take_screenshot('wifi')
-        settings.return_to_prev_menu(settings.screen_element)
+        # opening each subpage in Settings menu.
+        # some subpages have their own subpages, and they need to be opened as well.
+        ##################### Wi-Fi and its subpages ######################
+        wifi_page = settings.open_wifi()
+        Wait(self.marionette).until(lambda m: wifi_page.is_wifi_enabled is True)
+        
+        self.take_screenshot('wifi-enabled')
+        wifi_page.tap_connect_with_wps()
+        self.take_screenshot('wifi-wps')
+        settings.return_to_prev_menu(wifi_page.screen_element, wifi_page.wps_screen_element)
+        self.marionette.execute_script('arguments[0].scrollIntoView(false);', [wifi_page.manage_network_button])
```

Comment:
yeah, but if I don't do this, there is a subtle timing issue that would make the tapping unresponsive.  it will auto scroll to the area, but it won't tap.